pg_search/tests/pg_regress/expected/mixedff_advanced_03_limit_topn.out

```diff
@@ -446,30 +446,38 @@ EXPLAIN (FORMAT TEXT, COSTS OFF, TIMING OFF)
 SELECT title, is_available, rating
 FROM limit_topn_test
 WHERE is_available = true
-ORDER BY rating DESC
+ORDER BY rating DESC, title ASC
 LIMIT 7;
-               QUERY PLAN                
------------------------------------------
+                                           QUERY PLAN                                            
+-------------------------------------------------------------------------------------------------
  Limit
-   ->  Sort
-         Sort Key: rating DESC
-         ->  Seq Scan on limit_topn_test
-               Filter: is_available
-(5 rows)
+   ->  Incremental Sort
+         Sort Key: rating DESC, title
+         Presorted Key: rating
+         ->  Custom Scan (ParadeDB Scan) on limit_topn_test
+               Table: limit_topn_test
+               Index: limit_topn_idx
+               Exec Method: TopNScanExecState
+               Scores: false
+                  Sort Field: rating
+                  Sort Direction: desc
+                  Top N Limit: 7
+               Tantivy Query: {"term":{"field":"is_available","value":true,"is_datetime":false}}
+(13 rows)
 
 SELECT title, is_available, rating
 FROM limit_topn_test
 WHERE is_available = true
-ORDER BY rating DESC
+ORDER BY rating DESC, title ASC
 LIMIT 7;
    title    | is_available | rating 
 ------------+--------------+--------
+ Product 14 | t            |      5
+ Product 24 | t            |      5
+ Product 34 | t            |      5
+ Product 4  | t            |      5
  Product 44 | t            |      5
  Product 54 | t            |      5
```

Comment:
added an extra ORDER BY column here to ensure the results come back in a predictable order.  This query now gets pushed down so the results, ordered by rating, aren't necessarily preductable anymore.